.vale/styles/config/vocabularies/Data/accept.txt

```diff
@@ -0,0 +1,28 @@
+autoscaler
```

Comment:
Did we delete the old style pages (e.g., `.vale/styles/Vocab/Data/accept.txt`)?

Reply:
Thanks for catching! I didn't commit those changes. Just did it.